docs/ref/checks.txt

```diff
@@ -392,6 +392,11 @@ Models
 * **models.W045**: Check constraint ``<constraint>`` contains ``RawSQL()``
   expression and won't be validated during the model ``full_clean()``.
 
+Migrations
+----------
+
+...
+
```

Comment:
Please revert.

Reply:
We should [document](https://docs.djangoproject.com/en/stable/ref/checks/#builtin-tags) the `migrations` tag.